src/Router.ts

```diff
@@ -115,11 +115,9 @@ export class Router {
                     postProcessors.push(route);
                 }
                 if (route.handle) {
+                    handlingRoute = route;
```

Comment:
So much meaning in one line of code.

Reply:
What if none of the routes actually respond to the event and we get to the defaultRoute?  Should that be the `handlingRoute`?

Reply:
I would make a case for "yes": the array of handlingRoutes is useful for understanding what happened to the event, so if it gets all the way through to the defaultRoute, that's meaningful information. Or am I missing something?

Reply:
Updated & added test.